lib/Sema/TypeChecker.cpp

```diff
@@ -294,7 +294,6 @@ static void bindExtensions(SourceFile &SF) {
 
 static void typeCheckFunctionsAndExternalDecls(SourceFile &SF, TypeChecker &TC) {
```

Comment:
Looks like 'SF' isn't used anywhere in the body, so we can remove it from the parameter list

Reply:
I was hoping to remove this function altogether in #28138

Reply:
I was hoping to get rid of this function altogether in #28138.